doc/source/server_architecture.rst

```diff
@@ -0,0 +1,86 @@
+AIT Server Introduction
+========================
+
+The core of AIT is a "server", :mod:`ait.core.server`, which manages the processing of telemetry, logging, and command messages. The server manages relationships between the following components: 
+
+* **inbound streams**   - inbound streams manage incoming data being recieved by AIT, for example telemtry messages coming into a port.
```

Comment:
There's a few typos here. Updated along with slight format change:
```
Inbound streams manage incoming data being received by AIT. For example, telemetry messages coming into a port.
```

I think the same format change for outbound streams make sense as well. Similarly, capitalize all the list items helps clean it up a bit IMO.